understanding/21/pointer-cancellation.html

```diff
@@ -13,7 +13,9 @@ <h2>Intent</h2>
 			<h3>Up-Event activation or completion</h3>
 			<p>The most accessible way to incorporate pointer cancellation is to make activation occur on the up-event.</p>
       
-        <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed.</p>
+        <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed. Note however that the touchend event, if defined by the author, is fired at the original target element even when the pointer is released outside the target boundary.</p>
+      <p>In contrast, the down-event can occur as soon as the pointer is pressed inside a target. Whether this actually happens will depend on the mode of input. For example, the mousedown event will lead to a down-event down-event when using mouse or trackpad input, but it will not immediately fire when using touch input in the same setting. Using touch input, it only fires at the time of the up event. On devices with touch screens, a down-event will be triggered when authors use the touchstart event, while this event will not be fired when the same control is activated via mouse or trackpad input.</p>
```

Comment:
Again, quite complex, can it be simplified? If I’m understanding correctly, could we say something like:
> In contrast the down-events can occur as soon as the pointer is pressed inside a target. The mousedown and touchstart events can trigger immediately, there are some differences depending on the input device, but unless the content will only be used in one modality they should both be avoided.

Reply:
I think your sentence is a bit unclear. I am also not sure whether we should give authors the idea that web content may only be used on mobile devices (where apparently mousedown is queued to occur just before the mouseup touchend event). I think @patrickhlauke Patrick might be the best one to phrase this...

Reply:
Maybe ‘modality’ isn’t the right word then, it doesn’t mean just on mobile, it means in a mode (touch or mouse).

How about:
> In contrast the down-events can occur as soon as the pointer is pressed inside a target. The mousedown and touchstart events can trigger immediately.  There are some differences depending on the input device, but they should both be avoided for general web content.